tools/test/examples/examples_lib.py

```diff
@@ -349,7 +349,7 @@ def status(message):
     return results
 
 
-def compile_repos(config, toolchains, targets, profile, verbose, examples):
+def compile_repos(config, toolchains, targets, profile, verbose, examples, jobs=0):
```

Comment:
Need to add the new 'jobs' arg to the function documentation below.

Reply:
Seems that documentation need other fixes as well. Done.

Reply:
Ok, thanks